lms/djangoapps/ccx/views.py

```diff
@@ -346,6 +348,37 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
     )
 
 
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@coach_dashboard
+def update_course_details(request, course, ccx=None):
+    """
+    Update course display name for the CCX.
+    """
+    if not ccx:
+        raise Http404
+
+    name = request.POST['display_name']
+    CustomCourseForEdX.objects.filter(
+        course_id=course.id,
+        coach=request.user).update(display_name=name)
+    override_field_for_ccx(ccx, course, 'display_name', name)
+
+    # using CCX object as sender here.
+    responses = SignalHandler.course_published.send(
+        sender=ccx,
+        course_key=CCXLocator.from_course_locator(course.id, unicode(ccx.id))
+    )
+    for rec, response in responses:
+        log.info('Signal fired when course is published. Receiver: %s. Response: %s', rec, response)
```

Comment:
Nit: This isn't strictly necessary to log, but if you are logging, please first log the CCXLocator so we know which course was published.